src/sugar3/speech.py

```diff
@@ -212,6 +213,7 @@ def __init__(self, **kwargs):
         self._player.connect('play', self._update_state, 'play')
         self._player.connect('stop', self._update_state, 'stop')
         self._player.connect('pause', self._update_state, 'pause')
+        self._player.connect('mark', self._marked_messages_cb)
```

Comment:
Fit our code style.  Method name should be `__mark_cb`.

(`_update_state` doesn't fit our style, but no need to change it.)

Reply:
Will fix that, thanks.